debian/clearwater-snmpd.postinst

```diff
@@ -58,6 +58,7 @@ case "$1" in
         cp /etc/snmp/snmpd.conf.clearwater-snmpd /etc/snmp/snmpd.conf
 
         # Create the folder to store the statistics UNIX domain sockets.
+        rm -rf /var/run/clearwater
```

Comment:
snmpd doesn't create the stats sockets either so can't delete the stats directory. I am happy with you deleting the alarms directory if you can confirm that snmpd will be (re)started after this code has been executed (so won't end up bound to a socket that no longer exists).